auter.aptModule

```diff
@@ -73,10 +75,13 @@ function apply_updates() {
     # installed (i.e. dependencies of other packages). Instead we need to use install.
     UPDATEACTION="install"
   else
-    local ERROR_COUNT=$(${PACKAGE_MANAGER} -u upgrade --assume-no ${PACKAGEMANAGEROPTIONS} | grep -c '^[WE]:')
-    local AVAILABLE_PACKAGE_COUNT=$(${PACKAGE_MANAGER} -u upgrade --assume-no ${PACKAGEMANAGEROPTIONS} | awk '/upgraded,.*newly installed,/ {sum=$1+$3} END {print sum}')
+    local AVAILABLE_PACKAGES=$(${PACKAGE_MANAGER} -u upgrade --assume-no ${PACKAGEMANAGEROPTIONS})
```

Comment:
This will not capture error message, that are written into stderr.
Adding `2>&1` redirection  before closing `)` would allow proper detection of errors when calculating **ERROR_COUNT**  on line 80.